### PR TITLE
Add samples to drop all EMSG boxes from segment

### PR DESCRIPTION
**Pull request by mukulmishra18**
This PR add some sample code to drop all EMSG boxes present in the segment.

There is a limitation on the PS5 that there should not be more than 47 boxes before `mdat`. Having too many EMSG boxes can cause Media decode error.

### CODE REVIEW

File: ps5/dropAllConsecutiveEmsgBox.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  var bufferLevels = {};
+
+  bufferLevels['forwardduration'] = 12;
+  bufferLevels['backwardduration'] = 12,
+
+  bufferConfig['video'] = bufferLevels;
+  bufferConfig['audio'] = bufferLevels;
+
+  var conf = { 
+    key: '6c88f166-fe3e-467a-9276-aea43864cdcb',
```
Comment (protyposis) [inline ps5/dropAllConsecutiveEmsgBox.html]: Is this key allowed to be public?

Comment (mukulmishra18) [reply ps5/dropAllConsecutiveEmsgBox.html]: I think not. I am working on squashing the commits and force pushing to rewrite the history.

Comment (mukulmishra18) [reply ps5/dropAllConsecutiveEmsgBox.html]: Although I think we sometime share the code with this key to our customer.

**Review by protyposis — COMMENTED** [review]
History looks good now, but the key is still visible in my code comment 🤔

**Comment by mukulmishra18** [comment]
> but the key is still visible in my code comment 🤔

Maybe you can just delete the comment?

**Comment by protyposis** [comment]
Yap, had to delete all our comments.

**Review by wasp898 — APPROVED** [review]
Please also remove the NBA sources. Other than that, LGTM 👍

**Pull request merged**